source/more-info/unsupported/content_trust.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: "Ignored content trust"
+description: "More information on why disabling content-trust marks the installation as unsupported."
+---
+
+## The issue
+
+You have disable the content trust check. That means we can't veryify anymore if the content is trusted and was not modify by attackers.
+We strongly recomment to have this enabled.
+
+For Content-Trust, we use the OpenSource solution [CodeNotary](https://codenotary.io).
+
+## The solution
+
+To solve this you need to re-enable the protection mechanism by content-trust by using the CLI:
+
+```bash
+ha supervisor options --content-trust
+```
+
+When you have the content-trust, restart the Supervisor.
+
+```bash
+ha supervisor restart
+```
```

Comment:
```suggestion
---
title: "Ignored content trust"
description: "More information on why disabling content-trust marks the installation as unsupported."
---

## The issue

You have disabled the content-trust check. That means Home Assistant can't verify anymore if the content is trusted and was not modified by attackers.
We strongly recommend to have this check enabled.

For content-trust, we use the open source solution [CodeNotary](https://codenotary.io).

## The solution

To resolve this status, you need to re-enable the protection mechanism of content-trust by using the CLI:

```bash
ha supervisor options --content-trust
```

When you have enabled the content-trust, restart the Supervisor.

```bash
ha supervisor restart
```
```

Reply:
look like the suggestion is corrupt?

Reply:
I will delete and re-add